packages/body/tests/site/index.html

```diff
@@ -0,0 +1,125 @@
+		</script>
+	<![endif]-->
+</head>
+<body>
+	<noscript>
+		<p role="alert">This website needs JavaScript to work properly.</p>
```

Comment:
This entire file is just for local testing and will never be in any deliverable. I just copied it over from another project so we can focus on getting something out. ;)